### DISCUSSION TITLE
No. proteins: precursor vs protein levels

### DISCUSSION THREAD

**Discussion by haianhle** [vdemichev/DiaNN#476] [General]
Hi Vladim,

I was wondering what the difference is between the number of proteins at precursor level and protein level in the log output?

For example,
Number of proteins identified at 1% FDR: 7926 (precursor-level), 7737 (protein-level) (inference performed using proteotypic peptides only)

Thank you so much,
Anh

**Comment by vdemichev** [comment]
Hi Anh,

'Precursor-level' means without protein FDR filtering.

Best,
Vadim
